libs/frontend/feature-command/src/lib/component/replace-page-root.ts

```diff
@@ -0,0 +1,39 @@
+import { setSelectedComponent } from '@pubstudio/frontend/feature-editor'
+import { resolveComponent } from '@pubstudio/frontend/util-builtin'
+import { IReplacePageRootData } from '@pubstudio/shared/type-command-data'
+import { ISite } from '@pubstudio/shared/type-site'
+import { addComponentHelper, undoAddComponent } from './add-component'
+import { applyRemoveComponent, undoRemoveComponentHelper } from './remove-component'
+
+export const applyReplacePageRoot = (site: ISite, data: IReplacePageRootData) => {
```

Comment:
Hi @sampullman 

In the current implementation, the new component (pasted component) will be added to the site with `addComponentHelper`, so the `nextId` in site context will be incremented during replace, and will be decremented (restored) when undo. Do you think this make sense?